projects/fourier/lib/fourier/commands/bundle.rb

```diff
@@ -13,10 +13,39 @@ class Bundle < Base
         required: false,
         aliases: :p,
       )
+      option(
+        :xcode_version,
+        desc: %(
+          The version of Xcode to use to build tuist.
+          Can be either a version number (e.g. `--xcode_version="13.2.1"`)
+          or a file path pointing to the Xcode app bundle to use. (e.g. `--xcode_version="/path/to/Xcode.app"`)
+          Defaults to the currently selected Xcode on the system.
+        ),
+        type: :string,
+        required: false
+      )
+      option(
+        :xcode_version_libraries,
+        desc: %(
+          The version of Xcode to use to build tuist's library dependencies.
+          Can be either a version number (e.g. `--xcode_version="13.2.1"`)
+          or a file path pointing to the Xcode app bundle to use. (e.g. `--xcode_version="/path/to/Xcode.app"`)
+          Defaults to the currently selected Xcode on the system.
+        ),
+        type: :string,
+        required: false
+      )
       def tuist
         output_directory = options[:output]
         output_directory ||= File.expand_path("build", Constants::ROOT_DIRECTORY)
-        Services::Bundle::Tuist.call(output_directory: output_directory)
+        xcode_path = Utilities::Xcode.path_to_xcode(options[:xcode_version])
+        xcode_path_libraries = Utilities::Xcode.path_to_xcode(options[:xcode_version_libraries])
+
+        Services::Bundle::Tuist.call(
+          output_directory: output_directory,
+          xcode_path: xcode_path,
+          xcode_path_libraries: xcode_path_libraries
+        )
```

Comment:
Following @danyf90's advice here, I moved the parsing of the passed-in `xcode_version` arguments to happen right after they were passed in, and everything later on expects file paths.